appimagecraft.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
+
+project:
+  name: com.github.xournalpp.xournalpp
+  version_command: git describe --tags
```

Comment:
@Technius I should've mentioned that this command to detect the current version was just a random guess I made, doesn't seem to work all that well since I just tried and the app said its version is 1.0.11, whereas this command gave me 1.0.8-...
This should be fixed so that both the version noted in the AppImage and the one in the app's about dialog are the same.

Reply:
Looks like we only tagged the last two releases on separate branches, so the tags on master didn't get updated. Hopefully this should be fixed by the next release.

Easy workaround for now would be to set up cmake to write the version string to a file during builds and then read that file to get the version.